Deprecate trial (#17955)
cockroachdb/docs: src/current/v22.2/movr-flask-deployment.md

1. On the **Overview** page, select **Create Cluster**.

1. On the **Create new cluster** page:
    - For **Plan**, select CockroachDB {{ site.data.products.dedicated }}.
    - For **Cloud Provider**, select Google Cloud.
    - For **Regions & nodes**, add "us-east1", "us-west1", and "europe-west1", with 3 nodes in each region.

>     - Leave the **Hardware** and **Cluster name** as their default values.

Reviewer: ```suggestion
    - Use the default values for **Hardware** and **Cluster name**.
```

>     - For **Additional settings**, turn on VPC peering, with the default IP range.

Reviewer: ```suggestion
    - For **Additional settings**, enable VPC peering and use the default IP range.
```
